.idea/src/lesson15/HomeTask/Animal.java

```diff
@@ -0,0 +1,29 @@
+package lesson15.HomeTask;
+
+public class Animal {
+    public String TypeOfAnimal;
```

Comment:
variables and field names must start with a lower case. if You have class Animal  field can be just  - type and name instead of  TypeOfAnimal and nameOfAnimal